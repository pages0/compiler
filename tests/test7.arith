if 5 <= 5 then 5 <= 4 else (1 + 1) * (8 / 4)