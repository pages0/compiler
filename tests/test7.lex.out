(,if,(,<=,5,5,),(,<=,5,4,),(,*,(,+,1,1,),(,/,8,4,),),)